OpenRA.Mods.Common/Traits/Transforms.cs

```diff
@@ -96,7 +100,7 @@ public Order IssueOrder(Actor self, IOrderTargeter order, Target target, bool qu
 		public void DeployTransform(bool queued)
 		{
 			var building = self.TraitOrDefault<Building>();
-			if (!CanDeploy() || (building != null && !building.Lock()))
+			if (!CanDeploy(true) || (building != null && !building.Lock()))
```

Comment:
Actually, this requires a bigger change to handle the building locking... so maybe not a good idea for the prep branch.  This trait needs a total rewrite in the near future.